tests/integration/insights/test_category_import.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+
+from robotoff import settings
+from robotoff.insights.dataclass import InsightType
+from robotoff.insights.importer import InsightImporterFactory
+from robotoff.models import ProductInsight
+from robotoff.prediction.types import Prediction, PredictionType, ProductPredictions
+from robotoff.products import Product
+
+insight_id1 = "94371643-c2bc-4291-a585-af2cb1a5270a"
+barcode1 = "00001"
+
+
+@pytest.fixture(autouse=True)
```

Comment:
I would prefer not using `autouse=True`, as currently this fixture is run for all unit tests. Better including it manually as it is only useful for this module.

Reply:
I don't think so, it is only run for the test of this module.

See [the documentation](https://docs.pytest.org/en/latest/reference/fixtures.html#fixture-availability)
> Similarly, a test can also only be affected by an autouse fixture if that test is in the same scope that autouse fixture is defined in (see Autouse fixtures are executed first within their scope).

It would be true if fixture was in conftest.py

Reply:
I tested it running with a pdb, to be sure.